srcpkgs/python3-cysignals/template

```diff
@@ -1,18 +1,30 @@
 # Template file for 'python3-cysignals'
 pkgname=python3-cysignals
-version=1.11.3
+version=1.11.4
 revision=1
-build_style=python3-module
-hostmakedepends="python3-setuptools python3-Cython autoconf"
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel python3-Cython"
 makedepends="python3-devel pari-devel"
 depends="python3"
 short_desc="Interrupt and signal handling for Cython"
 maintainer="Gonzalo Tornaría <tornaria@cmat.edu.uy>"
 license="LGPL-3.0-or-later"
 homepage="https://github.com/sagemath/cysignals"
+changelog="https://github.com/sagemath/cysignals/releases"
 distfiles="${PYPI_SITE}/c/cysignals/cysignals-${version}.tar.gz"
-checksum=c2c01f666e5904948952a2250548d8c36ccedd640383791feacf26ea2b958365
-nocross=yes # runs binaries built for target
+checksum=0f1e321e55a07f901c86a36a1e4497f6ff9dfe700681d0130a38c36e4eb238c3
+
+# run autoconf after patching configure.ac
+hostmakedepends+=" autoconf"
+post_patch() {
```

Comment:
put this up with the other hostmakedepends and this should probably be pre_configure

Reply:
I put this in a block by itself since I hope it will be upstreamed soon and I'll just remove this, but using the other hostmakedepends is ok.

About where to run autoconf, I tend to think it belongs more to the patch step than to the configure step. I.e. I only need to run it once after patching so I consider it part of patch (e.g. upstream ships configure included in the tarball, so patch+autoconf is creating a source dir similar to what upstream ships).

No big deal either way.